### PR TITLE
BETA: Register baodev.is-a.dev

### PR DESCRIPTION
**Pull request by gibaohung-dev**
Added `baodev.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

**Comment by gibaohung-dev** [comment]
can anyone help me do this?

File: domains/baodev.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "gibaohung-dev",
+        "email": "thisismegiabao@gmail.com"
+    },
+    "record": {
+        "URL": "hosts.is-a.dev"
```
Comment (wdhdev) [inline domains/baodev.json:7]: is-a.dev Hosting was shutdown.

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/baodev.json (hunk 1 above)
Comment (gibaohung-dev) [reply domains/baodev.json:7]: so what can i do?

Comment (wdhdev) [reply domains/baodev.json:7]: Use GitHub Pages or another website hosting service to host a website on an is-a.dev subdomain.

**Pull request closed without merging**